### PR TITLE
Add script to export TorchScript model

### PR DESCRIPTION
**Pull request by strawberrypie**
This PR adds a script that creates a TorchScript model and, improves documentation and demo for LibTorch. Also, ONNX export script is renamed for clarity.

### CODE REVIEW

File: tools/export_torchscript.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        model.eval().cpu()
+        model_traced = torch.jit.trace(model, example_inputs=dummy_input).eval()
+        model_traced.save(output_path)
+        print('Finished export to TorchScript')
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        description='Convert .pth model weights to TorchMcript.')
```
Comment (zhiqwang) [inline tools/export_torchscript.py:36]: TorchMcript -> TorchScript, there may be a typo here?

File: demo_libtorch/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+python ./tools/export_torchscript.py --cfg_path ${CONFIG_PATH} --model_path ${PYTORCH_MODEL_PATH} --input_shape ${MO}
+```
+## Build
+
+### Linux
+```shell
+mkdir build
+cd build
+cmake -DCMAKE_PREFIX_PATH=/absolute/path/to/libtorch ..
+make --build . --config Release
```
Comment (RangiLyu) [inline demo_libtorch/README.md:31]: It should be like ```cmake --build . --config Release``` here or just ```make . ```

**Review by RangiLyu — CHANGES_REQUESTED** [review]

File: demo_libtorch/README.md (hunk 2 above)
Comment (strawberrypie) [reply demo_libtorch/README.md]: Thanks for the catch! I used PyTorch docs and they used `cmake` for both operations, but I'm used to `cmake` + `make`.

**Review by RangiLyu — APPROVED** [review]

**Comment by RangiLyu** [comment]
Thanks for your contribution!

**Pull request merged**